[2024-09-02T07:27:27.859+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T07:27:27.876+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:27:26.784342+00:00 [queued]>
[2024-09-02T07:27:27.885+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:27:26.784342+00:00 [queued]>
[2024-09-02T07:27:27.886+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T07:27:27.900+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-09-02 07:27:26.784342+00:00
[2024-09-02T07:27:27.908+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3209) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T07:27:27.909+0000] {standard_task_runner.py:72} INFO - Started process 3210 to run task
[2024-09-02T07:27:27.909+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cathcup_backfill_v1', 'task1', 'manual__2024-09-02T07:27:26.784342+00:00', '--job-id', '429', '--raw', '--subdir', 'DAGS_FOLDER/cathcup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpozbttur4']
[2024-09-02T07:27:27.910+0000] {standard_task_runner.py:105} INFO - Job 429: Subtask task1
[2024-09-02T07:27:27.956+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:27:26.784342+00:00 [running]> on host efc48c282cbe
[2024-09-02T07:27:28.037+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_cathcup_backfill_v1' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T07:27:26.784342+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T07:27:26.784342+00:00'
[2024-09-02T07:27:28.038+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T07:27:28.052+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T07:27:28.053+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-09-02T07:27:28.064+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T07:27:28.066+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-09-02T07:27:28.066+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T07:27:28.092+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T07:27:28.093+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_cathcup_backfill_v1, task_id=task1, run_id=manual__2024-09-02T07:27:26.784342+00:00, execution_date=20240902T072726, start_date=20240902T072727, end_date=20240902T072728
[2024-09-02T07:27:28.125+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T07:27:28.140+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-02T07:27:28.143+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
